package.json

```
@@ -14,8 +14,10 @@
       "typeorm": "0.3.9"
    },
    "scripts": {
+      "tsc": "tsc",
       "dev": "ts-node-dev --transpile-only --no-notify --exit-child server.ts",
       "start": "ts-node src/index.ts",
+      "build": "npm run tsc -- --esModuleInterop server.ts",
```

```suggestion
      "build": "tsc --esModuleInterop server.ts",
```

Não precisa do `npm run` dentro dos scripts do `package.json`, e os argumento de cli podem ser usados sem o `--` prévio aqui tbm.